docs/source/compression_preview/changes.rst

```diff
@@ -0,0 +1,121 @@
+Major Enhancement of Compression in NNI 3.0
+===========================================
+
+To bolster additional compression scenarios and more particular compression configurations,
+we have revised the compression application programming interface (API) in NNI 3.0.
+If you are a beginner to NNI Compression, you could bypass this document.
+Nonetheless, if you have employed NNI Compression before and want to try the latest Compression version,
+this document will help you in comprehending the noteworthy alterations in the interface in 3.0.
+
+
+New compression version import path:
+
+.. code-block:: python
+
+    # most new compression related, include pruners, quantizers, distillers, except new pruning speedup
+    from nni.contrib.compression.xxx import xxx
```

Comment:
still under `contrib`?

Reply:
Yes, according to the previous discussion, new compression is in contrib in 3.0 preview, and will replace it in the official version.